### DISCUSSION TITLE
[Improvement]: Missing Developer View In Higher Level ENV Settings

### DISCUSSION THREAD

**Discussion by hkd987** [coollabsio/coolify#3168] [Improvement Requests]
### Description

<img width="1501" alt="Screenshot 2024-05-03 at 9 12 09 AM" src="https://github.com/coollabsio/coolify/assets/6254949/e93ed2ab-5e22-4a5a-9200-e8eebdf7ac60">

See the above screen shot, having the developer view would be great, without it the higher level env settings are painful to use. 

### Minimal Reproduction (if possible, example repository)

n/a

### Exception or Error

n/a

### Version

4.0.0-beta.274

**Comment by andrasbacsai** [comment]
It is not a bug, it is missing, because I did not add it yet. 🙂

**Reply by sanirasheed** [reply]
when will it be added lol
